### ISSUE TITLE
Automatic creation of directory ?

### ISSUE THREAD

**Issue by ekareme** [spaceconcordia/space-commander#57]
Should the Net2Com library creates the GND_PIPES "/home/pipes/ground/" and CS1_PIPES "/home/pipes" directrories if they don't exist.

**Comment by SpaceShawn** [comment]
Yes it should. Can you please implement this in a new pull request? Thanks
